### DISCUSSION TITLE
TelegramLog - Channel not allowed

### DISCUSSION THREAD

**Discussion by IaZ001** [shayaantx/botdarr#109] [Q&A]
Hello,

I have just installed Botdarr using Docker Compose. My environment variables seem to be correct, but when I try to send commands to the channel, such as '!Help', I receive this message: "2023-Jul-04 11:10:52 AM [pool-5-thread-1] WARN  TelegramLog - Channel test not allowed, check properties file"

Can you help me resolve this issue?

**Comment by shayaantx** [comment]
question posted here https://github.com/shayaantx/botdarr/issues/110
